site/_posts/2022-04-01-fusionauth-announces-biotech.md

```diff
@@ -0,0 +1,57 @@
+description: A game-changing new factor of authentication.
+author: Dan Moore
+image: blogs/simplepass/fusionauth-releases-simplepass-header-image.png
+category: blog
+excerpt_separator: "<!--more-->"
+---
+
+FusionAuth is proud to announce a breakthrough in user security, BioTech&trade;. This new technology will help users around the world more easily secure their accounts and data.
+
+<!--more-->
+
+## The problem 
+
+It's well known that passwords alone are simply not enough to provide secure credentials. That's why many applications offer the ability to add additional factors of authentication, also known as multi-factor authentication (MFA).
+
+Per a [recent Microsoft post](https://www.microsoft.com/security/blog/2019/08/20/one-simple-action-you-can-take-to-prevent-99-9-percent-of-account-attacks/):
+
+> By providing an extra barrier and layer of security that makes it incredibly difficult for attackers to get past, MFA can block over 99.9 percent of account compromise attacks. 
+
+But for users, MFA is tedious. You have to set it up. You have to keep track of the additional factor, or rely on built-in services like FaceID and TouchID which require you to trust vendors like Apple and Google.
+ 
+## The solution
+
+The FusionAuth R&D team, with decades of experience in security and software development, saw a clear market opportunity to augment [traditional MFA methods](/learn/expert-advice/authentication/multi-factor-authentication) such as time based one-time password (TOTP), facial recognition, and sending a code via SMS with a new, innovative MFA method.
+
+"The feature is remarkably simple," said CTO Daniel DeGroff. "Rather than send you a code that requires you to keep track of a device or rely on iOS or Android to implement FIDO2 correctly, BioTech&trade; utilizes any bodily fluid to authenticate a user."
+
+By partnering with a number of medical records companies, FusionAuth allows users to provide any bodily fluid, such as saliva, sweat, or tears of frustration as an additional factor of authentication. Because this is advanced biometric identification, FusionAuth called it BioTech&trade;.
+
+It's simple to get started. All administrators have to do is enable the BioTech&trade; multi-factor method in the Tenant settings of their FusionAuth instance:
+
+{% include _image.liquid src="/assets/img/blogs/fusionauth-biotech/tenant-settings.png" alt="The FusionAuth tenant settings screen." class="img-fluid" figure=true %}
+
+Admins can use the API to configure this setting as well, of course.
+
+During the login process, users are prompted to provide a sample of any bodily fluid. They simply drip it in front of the camera and the soon-to-be patented BioTech&trade; algorithm analyzes it and authenticates the user.
+
+Founder Brian Pontarelli continued: "After completing the initial prototype we looked at each other in disbelief. It doesn't get any more satisfying! Everyone's cursed at their computer, now they can spit at it too."
```

Comment:
Do we want to leave a little more to the imagination here? 😉 

> now they can spit at it too, or you know... blow their nose, bleed... or.. (cough) other stuff.

Reply:
Hahah. I had the test recipient hint at ... other options, thought that was enough.